client/src/app/interfaces/task.ts

```diff
@@ -0,0 +1,8 @@
+export interface ITask {
```

Comment:
If you don't want to use a `shared` folder (which is fine), I would at least add a comment to both the server/client `ITask` files saying to make sure to keep these in sync!